libs/net/dal/Migrations/1.0.169/Up/PostUp/00-NotificationTemplate.sql

```diff
@@ -0,0 +1,64 @@
+DO $$
+BEGIN
+
+UPDATE public."notification_template"
+SET
+  "body" = '@inherits RazorEngineCore.RazorEngineTemplateBase<TNO.TemplateEngine.Models.Notifications.NotificationEngineContentModel>
+@using System
+@using System.Web
+@{
+  var subscriberAppUrl = @SubscriberAppUrl?.ToString();
+  var viewContentUrl = @ViewContentUrl?.ToString();
+  var requestTranscriptUrl = @RequestTranscriptUrl?.ToString();
+  var addToReportUrl = @AddToReportUrl?.ToString();
+  var isAV = @Content.ContentType == TNO.Entities.ContentType.AudioVideo;
+  var isTranscriptAvailable = isAV && !string.IsNullOrWhiteSpace(@Content.Body) && @Content.IsApproved;
+  var sourceCode = !string.IsNullOrEmpty(@Content.Source?.Code) ? @Content.Source.Code : @Content.OtherSource;
+  var body = isTranscriptAvailable ? @Content.Body : (isAV ? @Content.Summary : @Content.Body);
+  //  body = System.Text.RegularExpressions.Regex.Replace(body, @"\r\n?|\n", "<br/>");
+  var replaceWith = "<br/>";
+  body = body.Replace("\r\n", replaceWith).Replace("\n", replaceWith).Replace("\r", replaceWith);
+
+  var tz = TimeZoneInfo.FindSystemTimeZoneById("Pacific Standard Time");
+  var utcOffset = tz.GetUtcOffset(System.DateTime.Now).Hours;
+}
+@if (!string.IsNullOrWhiteSpace(@Content.Source?.Name))
+{
+  <div>@Content.Source.Name</div>
```

Comment:
Should this be the Legal name or the Friendly name?

Reply:
I thought it was the property `name`? based on `SourceDetails` component :
<FormikText label="Legal Name" name="name" tooltip="The full name of the source" required />